.github/workflows/ci.yml

```diff
@@ -21,4 +21,38 @@ jobs:
         with:
           otp-version: ${{matrix.otp_vsn}}
           rebar3-version: '3.14'
-      - run: rebar3 test
+      - run: |
+          rm -rf example/_checkouts  # Prevent long running analysis
```

Comment:
Even though this exception is in `hank`'s config., I'm not certain it's filtering the folders prior to listing them. Local analysis takes a while, but eventually finishes.

Reply:
It is not. Hank analyses everything, then filters out the results.

Reply:
Oh, Ok.